unumpy/_multimethods.py

```diff
@@ -1421,3 +1564,212 @@ def _vander_default(x, N=None, increasing=False):
 @all_of_type(ndarray)
 def vander(x, N=None, increasing=False):
     return (x,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def split(ary, indices_or_sections, axis=0):
+    return (ary,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def array_split(ary, indices_or_sections, axis=0):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=2),
+)
+@all_of_type(ndarray)
+def dsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=1),
+)
+@all_of_type(ndarray)
+def hsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=0),
+)
+@all_of_type(ndarray)
+def vsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+def _tile_default(A, reps):
+    if ndim(reps) == 0:
+        reps = (reps,)
+
+    prep = len(reps) - ndim(A)
+    if prep > 0:
+        for _ in range(prep):
+            A = expand_dims(A, 0)
+    elif prep < 0:
+        reps = tuple(itertools.repeat(1, abs(prep))) + reps
+
+    for axis, dim in enumerate(reps):
+        if not isinstance(dim, numbers.Integral):
+            dim = dim[()]
+        dim = operator.index(dim)
+        arrays = [A] * dim
+        if len(arrays) == 0:
+            return asarray(arrays, dtype=A.dtype)
+
+        A = concatenate(arrays, axis=axis)
+
+    return A
+
+
+@create_numpy(_first2argreplacer, default=_tile_default)
+@all_of_type(ndarray)
+def tile(A, reps):
+    return (A, reps)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def repeat(a, repeats, axis=None):
+    return (a,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def delete(arr, obj, axis=None):
+    return (arr,)
+
+
+def _insert_argreplacer(args, kwargs, dispatchables):
+    def replacer(arr, obj, values, axis=None):
+        return (dispatchables[0], obj, dispatchables[1]), {"axis": axis}
+
+    return replacer(*args, **kwargs)
+
+
+@create_numpy(_insert_argreplacer)
+@all_of_type(ndarray)
+def insert(arr, obj, values, axis=None):
+    return (arr, values)
+
+
+@create_numpy(_first2argreplacer)
+@all_of_type(ndarray)
+def append(arr, values, axis=None):
+    return (arr, values)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def resize(a, new_shape):
+    return (a,)
+
+
+def _trim_zeros_default(filt, trim="fb"):
+    nonzero_idxs = nonzero(filt)[0]
+    start, end = None, None
+    if "f" in trim:
+        start = nonzero_idxs[0]
+    if "b" in trim:
+        end = nonzero_idxs[-1] + 1
```

Comment:
You will need to special case the `len(nonzero_idxs) == 0` case where this would fail.